src/locales/en/messages.json

```diff
@@ -3031,5 +3031,14 @@
   },
   "masterPasswordPolicyRequirementsNotMet": {
     "message": "Your new master password does not meet the policy requirements."
+  },
+  "minimumNumberOfWords": {
+    "message": "Minimum Number of Words"
+  },
+  "passwordGeneratorPolicyDefaultType": {
```

Comment:
no need to prefix `passwordGeneratorPolicy` here. just say `defaultType`

Reply:
Resolved.